OJP/OJP_Trips.xsd

```diff
@@ -631,59 +631,219 @@
 			</xs:element>
 			<xs:element name="GuidanceAdvice" type="GuidanceAdviceEnumeration" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>various types of guidance advice given to travelle.</xs:documentation>
+					<xs:documentation>Various types of guidance advice given to traveller.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="TurnAction" type="TurnActionEnumeration" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>the range of alternative turns that can be described.</xs:documentation>
+					<xs:documentation>The range of alternative turns that can be described.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="RoadName" type="InternationalTextStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Road name, if applicable.</xs:documentation>
```

Comment:
Remove ", if applicable", the element has `minOccurs="0"`

Reply:
adapted